packages/material-ui/src/FilledInput/FilledInput.js

```diff
@@ -185,6 +185,7 @@ const FilledInput = React.forwardRef(function FilledInput(inProps, ref) {
     inputComponent = 'input',
     multiline = false,
     type = 'text',
+    hiddenLabel, // declare here to prevent leaking to DOM
```

Comment:
```suggestion
    hiddenLabel, // declare here to prevent spreading to DOM
```

"leaking" is usually used when talking about "leaking resources" e.g. memory leaks.